sp800-63b/sec5_authenticators.md

```diff
@@ -118,7 +118,7 @@ If the out-of-band authenticator sends an approval message over the secondary co
 
 ##### 5.1.3.2. Out-of-Band Verifiers
 
-Due to the risk that SMS messages or voice calls may be intercepted or redirected, implementers of new systems SHOULD carefully consider alternative authenticators. If the out-of-band verification is to be made using the public switched telephone network (PSTN), the verifier SHALL verify that the pre-registered telephone number being used is not associated with a VoIP (or other software-based) service. It then sends the SMS or voice message to the pre-registered telephone number. Changing the pre-registered telephone number SHALL NOT be possible without two-factor authentication at the time of the change.  **OOB using the PSTN (SMS or voice) is deprecated**, and may no longer be allowed in future releases of this guidance.
+Due to the risk that SMS messages or voice calls may be intercepted or redirected, implementers of new systems SHOULD carefully consider alternative authenticators. If the out-of-band verification is to be made using the public switched telephone network (PSTN), the verifier SHALL verify that the pre-registered telephone number being used is not associated with a VoIP (or other software-based) service. It then sends the SMS or voice message to the pre-registered telephone number. Changing the pre-registered telephone number SHALL NOT be possible without two-factor authentication at the time of the change.  **OOB using the PSTN (SMS or voice) is deprecated**, and is being considered for removal in future editions of this guideline.
```

Comment:
I like that idea. I'll add a commit to do that in a couple hours. Shall I merge then or would you like to do that?